features/borrowers_view_and_sign_deed.feature

```diff
@@ -0,0 +1,280 @@
+  Given I have valid deed data with <3> borrowers
+  And I create the deed via the Deed API
+  And borrower <2> views the deed
+  And borrower <1> views the deed again
+  Then borrower <3> views the deed again
+
+@us133
+Scenario: 3 Borrowers can view and sign a deed. p2
+  Given I have valid deed data with <3> borrowers
+  And I create the deed via the Deed API
+  And borrower <2> views the deed
+  Then the deed is digitally signed by borrower <2>
+  And borrower <3> views the deed again
+  Then I verify borrower <2> has signed the deed
+  Then the deed is digitally signed by borrower <3>
+  And borrower <1> views the deed again
+  Then I verify borrower <3> has signed the deed
+  Then I verify borrower <2> has signed the deed
+  Then the deed is digitally signed by borrower <1>
+
+@us133
+Scenario: 3 Borrowers can view and sign a deed. p3
+  Given I have valid deed data with <3> borrowers
+  And I create the deed via the Deed API
+  And borrower <2> views the deed
+  Then the deed is digitally signed by borrower <2>
+  And borrower <1> views the deed again
+  Then I verify borrower <2> has signed the deed
+  Then the deed is digitally signed by borrower <1>
+  And borrower <3> views the deed again
+  Then I verify borrower <1> has signed the deed
+  Then I verify borrower <2> has signed the deed
+  Then the deed is digitally signed by borrower <3>
+
+@us133
+Scenario: 3 Borrowers can view deed mutltiple times, and sign once. p1
+  Given I have valid deed data with <3> borrowers
+  And I create the deed via the Deed API
+  And borrower <1> views the deed
+  And borrower <2> views the deed again
+  Then the deed is digitally signed by borrower <2>
+  And borrower <3> views the deed again
+  Then I verify borrower <2> has signed the deed
+  Then the mortgage deed is displayed
+  And borrower <1> views the deed again
+  Then I verify borrower <3> has signed the deed
+  Then the deed is digitally signed by borrower <1>
+  And borrower <3> views the deed again
+  Then I verify borrower <1> has signed the deed
+  Then I verify borrower <2> has signed the deed
+  Then the deed is digitally signed by borrower <3>
+
+@us133
+Scenario: 3 Borrowers can view deed mutltiple times, and sign once. p2
+  Given I have valid deed data with <3> borrowers
+  And I create the deed via the Deed API
+  And borrower <3> views the deed
+  And borrower <1> views the deed again
+  Then the deed is digitally signed by borrower <1>
+  And borrower <2> views the deed again
+  And borrower <3> views the deed again
+  Then the deed is digitally signed by borrower <3>
+  And borrower <2> views the deed again
+  Then I verify borrower <3> has signed the deed
+  Then I verify borrower <1> has signed the deed
+  Then the deed is digitally signed by borrower <2>
+  And borrower <1> views the deed again
+  Then I verify borrower <1> has signed the deed
+  Then I verify borrower <2> has signed the deed
+  Then I verify borrower <3> has signed the deed
+
```

Comment:
As discussed with @nigelpain this is an important area, and high test coverage was decided as a good approach